Framework/PythonInterface/setup.py.in

```diff
@@ -0,0 +1,17 @@
+# -*- coding: utf-8 -*-
+#  This file is part of the mantid workbench.
```

Comment:
Small typo here. I think this file should actually have the short-form licence agreement at the top.